homeassistant/components/binary_sensor/wemo.py

```diff
@@ -17,11 +18,18 @@
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Register discovered WeMo binary sensors."""
     import pywemo.discovery as discovery
+    import requests
```

Comment:
`requests` is a core dependency of home assistant, so it should be imported at the top of the module.